apps/wakunode2/wakunode2_validator_signed.nim

```diff
@@ -49,21 +50,28 @@ proc withinTimeWindow*(msg: WakuMessage): bool =
     return true
   return false
 
-proc addSignedTopicValidator*(w: WakuRelay, topic: PubsubTopic, publicTopicKey: SkPublicKey) =
-  debug "adding validator to signed topic", topic=topic, publicTopicKey=publicTopicKey
+proc addSignedTopicsValidator*(w: WakuRelay, protectedTopics: seq[ProtectedTopic]) =
+  debug "adding validator to signed topics"
```

Comment:
Not a big deal but we are losing a bit of detail because we stopped logging the actual topic

Reply:
The issue here is that we changed the function so instead of adding one validator for a signed topic, it adds one validator function that works for all signed topics. This comes from the new approach of having the same validators running for all topics.

So I got rid of the topics parameter in the log because it's now a seq of structs and didn't know if it was worth it/spammy to print the whole sequence. I can 100% add it if useful, but the change comes that before it was easier/nicer because there was only one topic. Now it's a whole seq of structs.

Reply:
Yes makes sense, I think we can leave it as is. On the other hand, I think the "signed topics" come from a parameter and thus we could determine them if needed, AFAIK.